business_objects/comments.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime
+from submodules.model.business_objects import record
 from submodules.model.models import CommentData
```

Comment:
i think these should be relative paths (the CommentData one as well) since submodules don't know where they will be located